### ISSUE TITLE
Add BatchNorm Converter Fix / Replacement for new Partitioner

### ISSUE THREAD

**Issue by gs-olive** [pytorch/TensorRT#2120]
- Unify partitioner in export + compile paths

**Comment by gs-olive** [comment]
Completed in #2330
